inc/zoo/swar/SWAR.h

```diff
@@ -77,6 +77,25 @@ struct SWAR {
 
     constexpr T value() const noexcept { return m_v; }
 
+    // is this the right name?
+    constexpr static T MaxUnsignedLaneValue = ~(((~T{0}) << (NBits - 1)) << 1);
```

Comment:
in the standard librry, these things are called `UINT_MAX` and the like, they're macros.
We're not going to do that.
I much rather:
1. Have a descriptive name like "maximum value for an unsigned lane"
2. Make it a `T`, not a macro

This name satisfies the conditions, what's your potential objection to ask?